### ISSUE TITLE
Research Paper

### ISSUE THREAD

**Issue by shabu19** [keras-team/keras#14659]
Hi, is it necessary that your work accuracy must be greater then all the papers you have mentioned in literature review of your thesis. I'm working on HAR in keras.

**Comment by jbroot** [comment]
This isn't related to maintaining keras software.

**Comment by jvishnuvardhan** [comment]
@shabu19 As mentioned by @jbroot This is not related to bug/performance related issue. This repo is mainly for tracking the progress of bug/performance related issues.

This question is good for [TF Forum](https://discuss.tensorflow.org/tags) where other research express their take on your questions. Thanks!
